package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "precommit": "lerna run precommit",
-    "postinstall": "lerna bootstrap --loglevel verbose",
+    "postinstall": "lerna link && lerna exec -- npm i && lerna bootstrap --loglevel verbose",
```

Comment:
`lerna bootstrap` already runs `npm install` and links all the packages, no need to run them explictly.